src/Core/Location/PlusCodes/openlocationcode.cpp

```
@@ -0,0 +1,458 @@
+  // area.
+  double precision = compute_precision_for_length((int)code_length);
+  return latitude_degrees - precision / 2;
+}
+
+// Remove the separator and padding characters from the code.
+std::string clean_code_chars(const std::string &code) {
+  std::string clean_code(code);
+  clean_code.erase(
+      std::remove(clean_code.begin(), clean_code.end(), internal::kSeparator),
+      clean_code.end());
+  if (clean_code.find(internal::kPaddingCharacter)) {
```

Cppcheck (2.10-2)

[performance] Inefficient usage of string

(stlIfStrFind)